src/acc/libsmm_acc/libcusmm/collect.py

```diff
@@ -5,11 +5,26 @@
 import os
 from glob import glob
 import re
+import json
+from ast import literal_eval
+from kernels.cusmm_dnt import kernel_algorithm
 
 re_mnk    = re.compile("tune_(\d+)x(\d+)x(\d+)_")
 re_winner = re.compile("\nWINNER: \d+ (.+)\n")
 re_gflops = re.compile("# ([0-9.]+) GFlop/s")
 re_errors = re.compile("Number of errors: (\d+)\n")
+re_kernel_descr = re.compile("Kernel_dnt_(\w+)(\(.*\)) , # (\d+\.\d+) GFlop/s")
```

Comment:
regular expression strings should be prefixed with `r'...'` to avoid any accidental character interpretation